editor/js/Config.js

```diff
@@ -22,6 +22,8 @@ function Config() {
 		'project/renderer/physicallyCorrectLights': false,
 		'project/renderer/toneMapping': 0, // NoToneMapping
 		'project/renderer/toneMappingExposure': 1,
+		
+		'project/remoteScripts': 'https://code.jquery.com/jquery-3.5.1.min.js',
```

Comment:
I don't think this is a good default value. Many projects probably don't need jquery as a dependency.